docs/getting-started.md

```diff
@@ -13,7 +13,11 @@ We'll be going over the following steps:
 5. [Pushing new changes to the component](#5-pushing-new-changes-to-the-component)
 6. [Adding storage to the component](#6-adding-storage-to-the-component)
 
-**Extra documentation:**
+**Command completion**
+
+- [Command completion](#Command-completion)
```

Comment:
The clickable link doesn't seem to work for me

Reply:
@mik-dass Would you please let me know how you test it. However on mu macOS it works as expected. I use AsciiDoc in VS code editor to verify this.

Reply:
It _did_ work for me as well

Reply:
@amitkrout I viewed the file with "Display the rich diff" enabled on Github

Reply:
this works!
 but as @metacosm said this can be under extra docs

Reply:
@amitkrout Got it rich display doesn't work, I needed to do view file. Marking as resolved

Reply:
Awesome!!! I have been using ```AsciiDoc``` since a long time without any issue. You can try it out ;)